lxc/remote.go

```diff
@@ -24,18 +26,20 @@ func (c *remoteCmd) usage() string {
 	return gettext.Gettext(
 		"Manage remote LXD servers.\n" +
 			"\n" +
-			"lxc remote add <name> <url>        Add the remote <name> at <url>.\n" +
-			"lxc remote remove <name>           Remove the remote <name>.\n" +
-			"lxc remote list                    List all remotes.\n" +
-			"lxc remote rename <old> <new>      Rename remote <old> to <new>.\n" +
-			"lxc remote set-url <name> <url>    Update <name>'s url to <url>.\n" +
-			"lxc remote set-default <name>      Set the default remote.\n" +
-			"lxc remote get-default             Print the default remote.\n")
+			"lxc remote add <name> <url> [--password=PASSWORD]  Add the remote <name> at <url>.\n" +
+			"lxc remote remove <name>           	   	    Remove the remote <name>.\n" +
+			"lxc remote list                    	   	    List all remotes.\n" +
+			"lxc remote rename <old> <new>      	   	    Rename remote <old> to <new>.\n" +
+			"lxc remote set-url <name> <url>    	   	    Update <name>'s url to <url>.\n" +
+			"lxc remote set-default <name>      	   	    Set the default remote.\n" +
+			"lxc remote get-default             	   	    Print the default remote.\n")
 }
 
-func (c *remoteCmd) flags() {}
+func (c *remoteCmd) flags() {
+	gnuflag.StringVar(&c.password, "password", "", gettext.Gettext("Remote admin password"))
+}
 
-func addServer(config *lxd.Config, server string, addr string) error {
+func addServer(config *lxd.Config, server string, addr string, opts map[string]interface{}) error {
```

Comment:
Could those not just be additional arguments to the function? The casting seems not so nice.